pallets/stake/src/lib.rs

```diff
@@ -497,24 +520,32 @@ impl<T: Config> Module<T> {
 			if total == 0u32 {
 				return;
 			}
-			let issuance = T::Issuance::get();
+			let issuance = T::IssuancePerRound::get();
 			for (val, pts) in <AwardedPts<T>>::drain_prefix(round_to_payout) {
 				let pct_due = Perbill::from_rational_approximation(pts, total);
 				let mut amt_due = pct_due * issuance;
 				if amt_due < T::Currency::minimum_balance() {
 					continue;
 				}
 				if let Some(state) = <Candidates<T>>::get(&val) {
-					let fee_off_top = state.fee * amt_due;
-					if let Some(imb) = T::Currency::deposit_into_existing(&val, fee_off_top).ok() {
-						Self::deposit_event(RawEvent::Rewarded(val.clone(), imb.peek()));
-					}
-					amt_due -= fee_off_top;
-					for Bond { owner, amount } in state.nominators.0 {
-						let percent = Perbill::from_rational_approximation(amount, state.total);
-						let due = percent * amt_due;
-						if let Some(imb) = T::Currency::deposit_into_existing(&owner, due).ok() {
-							Self::deposit_event(RawEvent::Rewarded(owner.clone(), imb.peek()));
+					if state.nominators.0.len() == 1usize {
+						// solo validator with no nominators
```

Comment:
Technically, the validator is not a nominator of itself, but in theory its bond act as a nomination, right ?

Reply:
Yes, that is correct